shell/platform/embedder/embedder.h

```diff
@@ -1527,6 +1527,24 @@ typedef enum {
   kFlutterBackingStoreTypeSoftware2,
 } FlutterBackingStoreType;
 
+typedef struct {
+  /// The size of this struct. Must be sizeof(FlutterRegion).
+  size_t struct_size;
+  /// Number of rectangles in the region.
+  size_t rects_count;
+  /// The rectangles that make up the region.
+  FlutterRect* rects;
+} FlutterRegion;
+
+typedef struct {
```

Comment:
Similar comment here. This is really just for IDE hinting for folks who work against the embedder API; you've already documented the important stuff on the field.